src/Runners/PHPUnit/SuiteLoader.php

```diff
@@ -126,7 +127,11 @@ public function load(): void
     {
         $this->loadConfiguration();
 
+        $testSuiteCollection = null;
         if (($path = $this->options->path()) !== null) {
+            if (realpath($path) === false) {
```

Comment:
PHPUnit fails if provided path is not an existing file or directory, and we implement same behavior.